sys/linux/dev_snapshot.txt

```diff
@@ -0,0 +1,29 @@
+
+include <uapi/linux/suspend_ioctls.h>
+
+resource fd_snapshot[fd]
+
+openat$snapshot(fd const[AT_FDCWD], file ptr[in, string["/dev/snapshot"]], flags flags[open_flags], mode const[0]) fd_snapshot
+write$snapshot(fd fd_snapshot, buffer ptr[in, array[int8]], len bytesize[buffer])
+read$snapshot(fd fd_snapshot, buffer ptr[out, array[int8]], len bytesize[buffer])
+
+ioctl$SNAPHOT_FREEZE(fd fd_snapshot, cmd const[SNAPSHOT_FREEZE])
+ioctl$SNAPSHOT_UNFREEZE(fd fd_snapshot, cmd const[SNAPSHOT_UNFREEZE])
+ioctl$SNAPSHOT_CREATE_IMAGE(fd fd_snapshot, cmd const[SNAPSHOT_CREATE_IMAGE], arg ptr[in, int32])
+ioctl$SNAPSHOT_ATOMIC_RESTORE(fd fd_snapshot, cmd const[SNAPSHOT_ATOMIC_RESTORE])
+ioctl$SNAPSHOT_FREE(fd fd_snapshot, cmd const[SNAPSHOT_FREE])
+ioctl$SNAPSHOT_PREF_IMAGE_SIZE(fd fd_snapshot, cmd const[SNAPSHOT_PREF_IMAGE_SIZE])
```

Comment:
`SNAPSHOT_PREF_IMAGE_SIZE` uses the argument as an integer.

Reply:
`ioctl$SNAPSHOT_PREF_IMAGE_SIZE(fd fd_snapshot, cmd const[SNAPSHOT_PREF_IMAGE_SIZE], arg intptr)`